content/en/platform/corda/5.0-dev-preview-1/corda-cli/commands.md

```diff
@@ -597,3 +597,14 @@ alice: Node smoke-tests-network/alice was deployed with [solar-system.cpb]
 bob: Node smoke-tests-network/bob was deployed with [solar-system.cpb]
 Network smoke-tests-network was deployed with [solar-system.cpb]
 ````
+
+### Removing a CorDapp
+
+{{< note >}}
+It is not currently possible to remove a CorDapp from the network. Instead, you can terminate the network and restart.
+{{< /note >}}
```

Comment:
i would move this note to the end under the actual steps below so that the reading flow is not interrupted.

Reply:
The note is more urgent than the steps. Flow intentionally interrupter.